gradle.properties

```diff
@@ -1,8 +1,8 @@
 org.gradle.caching=true
 group=io.ballerina.stdlib
-version=1.5.1-SNAPSHOT
+version=1.6.0-SNAPSHOT
 
 ballerinaGradlePluginVersion=1.0.0
 
-ballerinaLangVersion=2201.3.0
-stdlibIoVersion=1.3.1
+ballerinaLangVersion=2201.4.0-20221116-082400-3eca35ee
```

Comment:
Shall we change this to `2201.4.0-20221208-193400-e9fb14ee`? That is the latest

Reply:
Updated in [e024dc1](https://github.com/ballerina-platform/module-ballerina-os/pull/476/commits/e024dc15b5de5944a893e8289394a8e4528b3737)